src/styles/theme/oc-button.scss

```diff
@@ -39,17 +69,83 @@
 
   &.uk-button-large {
     min-height: $global-control-large-height;
+    font-size: 1.5rem;
   }
 
   &.uk-button-primary,
   &.uk-button-secondary,
   &.uk-button-danger {
+    border: none;
+
     .oc-icon > svg {
       fill: $global-inverse-color;
     }
   }
+
+  &.uk-button-default {
+    border: 1px solid $action-primary;
+    background-color: transparent;
+    color: $action-primary;
+
+    &:hover,
+    &:focus {
+      border-color: $action-primary-hover;
+      color: $action-primary-hover;
+
+      .oc-icon > svg {
+        fill: $action-primary-hover;
+      }
+    }
+
+    .oc-icon > svg {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".oc-button.uk-button-default .oc-icon > svg" to come before selector ".oc-button.uk-button-default:hover .oc-icon > svg" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)